Dockerfile

```diff
@@ -47,17 +47,20 @@ COPY requirements.txt /app/requirements.txt
 
 # Install Python packages from requirements.txt
 RUN mamba install --yes --file requirements.txt && mamba clean --all -f -y
-RUN pip install rank_bm25 streamlit-pdf-viewer
+RUN pip install rank_bm25 streamlit-pdf-viewer jupyter
```

Comment:
Take out jupyter from here. We don't need this since it's present in our requirements.txt